.github/workflows/washboard.yml

```diff
@@ -0,0 +1,68 @@
+name: washboard
```

Comment:
could you also update `wash.yml` accordingly to remove the UI build?

Reply:
Was going to, but it seems more related to the work that will be completed in #875

Reply:
Yep I removed in https://github.com/wasmCloud/wasmCloud/pull/919